### PR TITLE
{math}[foss/2021b,intel/2021b] ELPA v2021.11.001

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
skylake-alma8-eb - Linux AlmaLinux 8.5, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/499adeb4fd52b8f98cba3ffb3619cdbd for a full test report.

File: easybuild/easyconfigs/e/ELPA/ELPA-2021.11.001-intel-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ('Autotools', '20210726'),
+    # remove_xcompiler script requires 'python' command,
+    ('Python', '3.9.6'),
+    ('Perl', '5.34.0'),
+]
+
+preconfigopts = './autogen.sh && '
+preconfigopts += 'export LDFLAGS="-lm $LDFLAGS" && '
+preconfigopts += 'autoreconf && '
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/e/ELPA/ELPA-2021.11.001-intel-2021b.eb:37]: the `intel/2021a` easyconfig has 

```python
# If the build is running in a batch job, make sure Intel libmpi does
# not try to use Slurm to run the MPI_THREAD test code that ELPA's
# configure is running without "mpirun".
preconfigopts += 'unset SLURM_JOBID && unset SLURM_NODELIST && unset I_MPI_PMI_LIBRARY && '
```
is this no longer relevant?

Comment (SebastianAchilles) [reply easybuild/easyconfigs/e/ELPA/ELPA-2021.11.001-intel-2021b.eb:41]: Thanks for catching this one. :+1: It is still relevant. Added in https://github.com/easybuilders/easybuild-easyconfigs/pull/14964/commits/95a61815a58f5dc92a671fbc17517d622f9671c0

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**